VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsBaseFragment.java

```diff
@@ -63,10 +63,9 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 DiscussionThread thread = discussionPostsAdapter.getItem(position);
                 router.showCourseDiscussionResponses(context, thread, courseData);
 
-                new SetThreadReadTask(context, thread, true).execute();
                 if (!thread.isRead()) {
                     // Refresh the row to mark it as read immediately,
```

Comment:
If the sentence is ended here, then there should be a period instead of comma at the end. Comment line length should also be normalized.